labgrid/driver/powerdriver.py

```diff
@@ -239,22 +239,31 @@ class YKUSHPowerDriver(Driver, PowerResetMixin, PowerProtocol):
     delay = attr.ib(default=2.0, validator=attr.validators.instance_of(float))
 
 
+    # use ykushcmd as control driver, because this is more convenient
+    # than a python module and supports remote control via ssh.
+    # ref: https://www.yepkit.com/learn/setup-guide-ykush
     def __attrs_post_init__(self):
         super().__attrs_post_init__()
```

Comment:
```suggestion
    def __attrs_post_init__(self):
        super().__attrs_post_init__()
        if self.target.env:
            self.tool = self.target.env.config.get_tool('ykushcmd') or 'ykushcmd'
        else:
            self.tool = 'ykushcmd'

```
Let's make the CLI tool configurable. Now please use `self.tool` instead of `ykushcmd` everywhere.